wiki/People/Performance_Points_Committee/id.md

```diff
@@ -0,0 +1,24 @@
+---
+stub: true
+tags:
+  - pp committee
+  - komite pp
+---
+
+# Performance Points Committee
+
+**Performance Points Committee** (atau *Komite Performance Points*) merupakan sebuah panel beranggotakan para pemain yang berpengalaman dan pengembang yang memfasilitasi dan menyetujui perubahan pada algoritma [performance points](/wiki/Performance_points) dan [star rating](/wiki/Beatmapping/Star_rating). Tujuan dari komite ini adalah untuk membantu para [pengembang osu!](/wiki/People/The_Team/Developers) dalam merumuskan algoritma perhitungan [performance point](/wiki/Performance_points) dan [star rating](/wiki/Beatmapping/Star_rating) yang akurat dengan mempertimbangkan berbagai masukan dari komunitas.
```

Comment:
No need to link twice, but can't the second sentence be expressed more concisely anyway?

```suggestion
**Performance Points Committee** (atau *Komite Performance Points*) merupakan sebuah panel beranggotakan para pemain yang berpengalaman dan pengembang yang memfasilitasi dan menyetujui perubahan pada algoritma [performance points](/wiki/Performance_points) dan [star rating](/wiki/Beatmapping/Star_rating). Tujuan dari komite ini adalah untuk membantu para [pengembang osu!](/wiki/People/The_Team/Developers) dalam merumuskan algoritma perhitungan performance point dan star rating yang akurat dengan mempertimbangkan berbagai masukan dari komunitas.
```

Reply:
Let's try to go back to the first change.